pkg/operator/operator/k8s.go

```diff
@@ -971,6 +1006,28 @@ func getEnvVars(api *spec.API, container string) []kcore.EnvVar {
 	return envVars
 }
 
+func getKubexitEnvVars(containerName string, deathDeps ...string) []kcore.EnvVar {
+	envVars := []kcore.EnvVar{
+		{
+			Name:  "KUBEXIT_NAME",
+			Value: containerName,
+		},
+		{
+			Name:  "KUBEXIT_GRAVEYARD",
```

Comment:
And these `KUBEXIT_*` vars are specific to what `kubexit` is expecting, right?

Reply:
yes, they are